src/api-guides/e-receipts.markdown

```diff
@@ -1,173 +1,159 @@
-- [One-Time Password Grant](/api-reference/authentication/apidoc.html#otp_grant) – Connect from the Partner App utilizing Email.
-
-
-#### App Center Flow - Connecting from Concur Web / Mobile App Center
-
-1. To begin the authentication flow, a request (temporary) token is obtained via the Concur AppCenter interface.
-2. A logged-in Concur user clicks on the Connect button on the App listing in the App Center and authorizes your app to post data.   App Listing Example below:
+1. To begin the authentication flow, a request (temporary) token is obtained via the SAP Concur AppCenter interface.
+2. A logged-in SAP Concur user clicks on the Connect button on the App listing in the App Center and authorizes your app to post data.   App Listing Example below:
 
     ![appflow_listing](/assets/img/api-guides/e-receipts/appflow_listing.png)
 
-3. Concur&#39;s authorization service will redirect the user to your landing page.  Please follow the [App Center Design Guidelines](/manage-apps/go-market-docs/app-center-ux-guidelines-consumer.html) to create a web page that listens for an HTTP GET request from Concur.
-4. The redirect URI will contain an **id** and **requestToken** parameters.  
+3. The SAP Concur authorization service will redirect the user to your landing page. Please follow the [App Center Design Guidelines](/manage-apps/go-market-docs/app-center-ux-guidelines-consumer.html) to create a web page that listens for an HTTP GET request from SAP Concur.
+4. The redirect URI will contain an `id` and `requestToken` parameters.  
 Example:   https://{partner_redirect_URI}?id=8568a4cd-8ffc-49d6-9417-be2d69aa075f&requestToken=5l85ae5a-426f-4d6f-8af4-08648c4b696b
-5. When your application receives the redirect call strip the **id** and **requestToken** values from the URI and use those on a Post request to the Concur Authorization service to obtain the official oauth2 accesstoken and refreshtoken for the user using the [password grant](/api-reference/authentication/apidoc.html#password_grant) while [being Geo Aware](#being_geo_aware).
-6. Decode the **id\_token** to obtain the **sub** value and store this value as the user **id** (see [https://jwt.io](https://jwt.io))
-7. An access token is valid only for one hour.  The access token should be cached in memory and discarded after use.
-8. After the user has successfully completed the login/enrollment process, store the following with the user&#39;s profile in your database.
+5. When your application receives the redirect call strip the `id` and `requestToken` values from the URI and use those on a Post request to the SAP Concur Authorization service to obtain the official OAuth2 `accessToken` and `refreshToken` for the user using the [password grant](/api-reference/authentication/apidoc.html#password_grant) while [being geo aware](#being-geo-aware).
+6. Decode the `id\_token` to obtain the `sub` value and store this value as the user `id` (see [https://jwt.io](https://jwt.io)).
```

Comment:
I'm not sure I understand the `\_` here -- I don't think it's necessary and the documentation this is based upon doesn't use it at all.